jira-macro/jira-macro-default/src/test/resources/macrojira7.test

```diff
@@ -13,7 +13,7 @@ XWIKI-1001
 beginDocument
 beginMacroMarkerStandalone [jira] [url=http://localhost:8889|style=enum] [XWIKI-1000
 XWIKI-1001]
-beginGroup
+beginParagraph
```

Comment:
I think we should update the test description to mention that it's testing the enum displayer when used in a standalone jira macro.